R/bootstrap.nlsfit.R

```diff
@@ -357,16 +375,16 @@ bootstrap.nlsfit <- function(fn,
   ## BUT it always has the same name
   if(errormodel == "yerrors"){
     if(useCov){
-      fitchi <- function(y, par, ...) { dY %*% (y - fn(par=par, x=x, ...)) }
+      fitchi <- function(y, par, ...) { W %*% (y - c(fn(par=par, x=x, ...), par[priors$param])) }
     }else{
-      fitchi <- function(y, par, ...) { dY * (y - fn(par=par, x=x, ...)) }
+      fitchi <- function(y, par, ...) { W * (y - c(fn(par=par, x=x, ...), par[priors$param])) }
     }
   }else{
     ipx <- length(par.Guess)-seq(nx-1,0)
     if(useCov){
-      fitchi <- function(y, par, ...) { dY %*% (y - c(fn(par=par[-ipx], x=par[ipx], ...), par[ipx])) }
+      fitchi <- function(y, par, ...) { W %*% (y - c(fn(par=par[-ipx], x=par[ipx], ...), par[ipx])) }
```

Comment:
This does not look correct for the xy-error case. But okay, we don't have to have all features right away. We should just make it fail in that case. So please add a `stop` near the top of the function that checks that the priors are not being used with the xy-error model and tells the user that this is not supported at the moment.